guides/introduction/negotiation.md

```diff
@@ -2,31 +2,29 @@
 
 Before starting to send or receive media, you need to negotiate the WebRTC connection first, which comes down to:
```

Comment:
Really nice! Clean and conciese :+1: 

The only thing is that I feel like we have too many admonitions in the `Offer and answer exchange section`. They distract  a  reader, don't they?

I am wondering about adding at its end something like this:

```
Notes:
* here admonition about `PeerConnection can be bidirectional`
* here admonition about `Renegotiations`
```

I think I would also inline `Offers, answers and SDP` admoniton and refactor it just a little to mention that answer  can reject incoming offer or its parts?

```
Finally, we have to create and set an offer.
It will contain information on how many tracks we want to send, which codecs we want to use, whether we are willing to also receive something or not and so on.
The other side responds with an answer, which can either accept, reject or partially accept our offer (e.g. accept only audio tracks).
Both offer and answer are carried in a text format called SDP. You can read more about it in the SDP Anatomy tutorial from webrtcHacks.
```

Any thoughts?

Reply:
I inlined the `Offers, answers and SDP` and `Renegotations`.